docs/template_plugin/tests/functional/subgraph_reference/preprocess.cpp

```diff
@@ -24,6 +24,8 @@ struct RefPreprocessParams {
         std::function<std::shared_ptr<ov::Function>()> function;
         std::vector<Tensor> inputs;
         std::vector<Tensor> expected;
+        float abs_threshold = 0.01f;
+        float rel_threshold = 0.01f;
```

Comment:
Why do we have so big default threshold?

Reply:
This default value is same as for 'base_reference_test'. https://github.com/openvinotoolkit/openvino/blob/acb6aa01ab97484e222ce77051c3015a546ab313/docs/template_plugin/tests/functional/op_reference/base_reference_test.hpp#L35